### DISCUSSION TITLE
Endir reglunnar - 2024/3/1

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#203] [vefsida-ummæli]
# Endir reglunnar - 2024/3/1

Heimasíða Biskupsreglunnar

https://www.biskupsreglan.com/frettir/Endir-reglunnar

**Comment by MatthiasGudni** [comment]
Ansans 🕊️

**Reply by TJ478** [reply]
Þetta eru hörmungar
